File: algorithms/solutions/week30/duythinht/mini-max-sum.c
#include <stdio.h>

void miniMaxSum(int n, int* a) {
    long sum = 0, min = a[0], max = a[0];
    for (int i=0; i < n; i++) {
        sum += a[i];
        if (a[i] < min) {
            min = a[i];
        } else if (a[i] > max){
            max = a[i];
        }
    }
    printf("%ld %ld ", sum - max, sum - min);
}

File: algorithms/solutions/week30/unrealhoang/mini-max-sum.clj
; Complete the miniMaxSum function below.
(defn miniMaxSum [arr]
    (let [sum (apply + arr)
          min (apply min arr)
          max (apply max arr)
          min-sum (- sum max)
          max-sum (- sum min)]
        (print min-sum max-sum))
)
